<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/Icons/Qt.png</normaloff>:/icons/Icons/Qt.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="ModelView" name="modelView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionAddOutputStream"/>
   <addaction name="actionAddFIFO"/>
   <addaction name="actionAddLIFO"/>
   <addaction name="actionAddSIRO"/>
   <addaction name="actionAddDevice"/>
   <addaction name="actionAddInputStream"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Icons/new.png</normaloff>:/icons/Icons/new.png</iconset>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
   <property name="toolTip">
    <string>Очистить сцену</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Icons/Play-1-Hot-icon.png</normaloff>:/icons/Icons/Play-1-Hot-icon.png</iconset>
   </property>
   <property name="text">
    <string>Начать</string>
   </property>
   <property name="toolTip">
    <string>Начать моделирование</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Icons/Pause-icon.png</normaloff>:/icons/Icons/Pause-icon.png</iconset>
   </property>
   <property name="text">
    <string>Пауза</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Icons/stop-red-icon.png</normaloff>:/icons/Icons/stop-red-icon.png</iconset>
   </property>
   <property name="text">
    <string>Стоп</string>
   </property>
   <property name="toolTip">
    <string>Сброс</string>
   </property>
  </action>
  <action name="actionAddOutputStream">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Icons/box-out-icon.png</normaloff>:/icons/Icons/box-out-icon.png</iconset>
   </property>
   <property name="text">
    <string>Исходящий поток</string>
   </property>
   <property name="toolTip">
    <string>Исходящий поток</string>
   </property>
  </action>
  <action name="actionAddFIFO">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Icons/FIFO.png</normaloff>:/icons/Icons/FIFO.png</iconset>
   </property>
   <property name="text">
    <string>Очередь FIFO</string>
   </property>
   <property name="toolTip">
    <string>Добавить очередь FIFO</string>
   </property>
  </action>
  <action name="actionAddLIFO">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Icons/LIFO.png</normaloff>:/icons/Icons/LIFO.png</iconset>
   </property>
   <property name="text">
    <string>Очередь LIFO</string>
   </property>
   <property name="toolTip">
    <string>Добавить очередь LIFO</string>
   </property>
  </action>
  <action name="actionAddSIRO">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Icons/SIRO.png</normaloff>:/icons/Icons/SIRO.png</iconset>
   </property>
   <property name="text">
    <string>Очередь SIRO</string>
   </property>
   <property name="toolTip">
    <string>Добавить очередь SIRO</string>
   </property>
  </action>
  <action name="actionAddDevice">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Icons/Norton-System-Works-icon.png</normaloff>:/icons/Icons/Norton-System-Works-icon.png</iconset>
   </property>
   <property name="text">
    <string>Прибор</string>
   </property>
   <property name="toolTip">
    <string>Добавить прибор</string>
   </property>
  </action>
  <action name="actionAddInputStream">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Icons/box-in-icon.png</normaloff>:/icons/Icons/box-in-icon.png</iconset>
   </property>
   <property name="text">
    <string>Замыкающий поток</string>
   </property>
   <property name="toolTip">
    <string>Добавить замыкающий поток</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>О Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModelView</class>
   <extends>QWidget</extends>
   <header>modelview.h</header>
   <container>1</container>
   <slots>
    <slot>addOutputStream()</slot>
    <slot>addSIRO()</slot>
    <slot>addLIFO()</slot>
    <slot>addFIFO()</slot>
    <slot>addDevice()</slot>
    <slot>addInputStream()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddDevice</sender>
   <signal>triggered()</signal>
   <receiver>modelView</receiver>
   <slot>addDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddFIFO</sender>
   <signal>triggered()</signal>
   <receiver>modelView</receiver>
   <slot>addFIFO()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddLIFO</sender>
   <signal>triggered()</signal>
   <receiver>modelView</receiver>
   <slot>addLIFO()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddOutputStream</sender>
   <signal>triggered()</signal>
   <receiver>modelView</receiver>
   <slot>addOutputStream()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddInputStream</sender>
   <signal>triggered()</signal>
   <receiver>modelView</receiver>
   <slot>addInputStream()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddSIRO</sender>
   <signal>triggered()</signal>
   <receiver>modelView</receiver>
   <slot>addSIRO()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
